modules/multi-packit.nix

```
@@ -74,6 +76,7 @@ in
 
         apiRoot = "https://${cfg.domain}/${name}/packit/api";
         outpackServerUrl = "http://127.0.0.1:${toString ports."${name}".outpack}";
+        orderlyRunnerApiUrl = "http://127.0.0.1:${toString ports.orderly-runner-api}";
```

```suggestion
        orderlyRunnerApiUrl = "http://127.0.0.1:${toString config.services.orderly-runner.port}";
```

See my comment earlier.

---

done

---

Huh turns out `config` here has been shadowed by the submodule's config. Looks like you might need to do a `let globalCfg = config` or something at the toplevel. A little ugly but heh

---

Alternatively, you change the top of this block from `"${name}" = ({ config, ... }: ` to `"${name}" = submodule:`, and then you replace the one place where we actually want the nested `config` by `submodule.config`. Arguably, I think this is cleaner than shadowing.